docker/crabserver/addGH.sh

```diff
@@ -13,9 +13,16 @@ CRABServerDir=`realpath /data/srv/current/sw*/*/cms/crabserver/*`
 # 1. find which GitHub tags were installed
 # beware that directory name in $CRABServerDir may not be the GH tag but rather
 # name assigned at build time like v3.230303-comp4
-export PYTHONPATH=${CRABServerDir}/lib/python2.7/site-packages
-CRABServerGHTag=`python -c "from CRABInterface import __version__; print __version__"`
-WMCoreGHTag=`python -c "from WMCore import __version__; print __version__"`
+source `realpath /data/srv/HG*/apps/crabserver/etc/profile.d/init.sh`
```

Comment:
Dario, can we avoid sourcing init.sh and this "if" by simply doiing some grep in `${CRABServerDir}/lib/python*/site-packages` ?

Reply:
I like this option! the new commit contains this approach